d2networking/d2client/d2localclient/local_client_connection.go

```diff
@@ -47,7 +49,7 @@ func Create(openNetworkServer bool) *LocalClientConnection {
 }
 
 // Open creates a new GameServer, runs the server and connects this client to it.
-func (l *LocalClientConnection) Open(connectionString string, saveFilePath string) error {
+func (l *LocalClientConnection) Open(_ string, saveFilePath string) error {
```

Comment:
Can we refactor this a bit in order to remove this parameter as it seems we don't really need it?

Reply:
The parameter is necessary to satisfy an interface, hence why I've left it with the blank identifier.

Reply:
Possibly something that warrants a comment, I initially removed the parameter myself.